subiquity/server/ubuntu_drivers.py

```diff
@@ -168,6 +170,98 @@ async def list_oem(self, root_dir: str, context) -> List[str]:
         return self._oem_metapackages_from_output(result.stdout.decode("utf-8"))
 
 
+class UbuntuDriversFakePCIDevicesInterface(UbuntuDriversInterface):
+    """An implementation of ubuntu-drivers that wraps the calls with
+    the umockdev wrapper script.
+
+
+    Requires an online install to download umockdev packages or
+    a modified ISOs with the packages added to the pool.
+    """
+
+    def __init__(self, app, gpgpu: bool) -> None:
+        super().__init__(app, gpgpu)
+
+        self.wrapper_script_source: str = umockdev_wrapper.script
+        self.script_path = "/tmp/umockdev_wrapper.py"
+
+        self.dev_config = configs.broadcom
```

Comment:
This is a bit hardcoded which means that we can't actually use the other config blobs, such as dell_certified_with_nvidia, without source modification.  Thoughts?

Reply:
I'm thinking with moving away from the system_scripts module, we have a default config like:
```python
        self.dev_config = {
            "devices": [
                {
                    "modalias": "pci:v000014E4d00004353sv00sd01bc02sc80i00",
                    "vendor": "0x14E4",
                    "device": "0x4353",
                }
            ]
        }
```
which gets written on boot to a known location (i.e. `/tmp/umockdev_config.yaml`) and then we just modify that with early-commands or similar before continuing with the install. This is also the inspiration for 1d5654510893cc5813da6d72113e359c09090cb4.  What do you think?